qiskit/__init__.py

```diff
@@ -36,23 +35,29 @@
 import qiskit.extensions.standard
 import qiskit.extensions.quantum_initializer
 
+import qiskit.backends.ibmq
+import qiskit.backends.local
+
 # Allow extending this namespace. Please note that currently this line needs
 # to be placed *before* the wrapper imports or any non-import code.
 __path__ = pkgutil.extend_path(__path__, __name__)
 
-# Allow extending this namespace. Please note that currently this line needs
-# to be placed *before* the wrapper imports.
-__path__ = pkgutil.extend_path(__path__, __name__)
+from .wrapper._wrapper import (compile, execute, load_qasm_string,
+                               load_qasm_file, least_busy, qobj_to_circuits)
+
+# To be deprecated methods
+from .wrapper._wrapper import (available_backends, get_backend, register,
+                               unregister, registered_providers)
 
-from .wrapper._wrapper import (
-    available_backends, get_backend,
-    compile, execute, register, unregister,
-    registered_providers, load_qasm_string, load_qasm_file, least_busy,
-    store_credentials)
 
 # Import the wrapper, to make it available when doing "import qiskit".
 from . import wrapper
 
+# Global objects. They are assigned to the specific module-level variables to
+# minimize the risk of them falling out of sync - please use with care.
+IBMQ = qiskit.backends.ibmq.IBMQ
+Aer = qiskit.backends.local.Aer  # pylint: disable=invalid-name
```

Comment:
Do we want to add this now? Probably should wait until we actually have aer on pypi.